code/datums/station_traits/negative_traits.dm

```diff
@@ -0,0 +1,92 @@
+/datum/station_trait/carp_infestation
+	name = "Carp infestation"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Dangerous fauna is present in the area of this station"
+	trait_to_give = STATION_TRAIT_CARP_INFESTATION
+
+/datum/station_trait/distant_supply_lines
+	name = "Distant supply lines"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 3
+	show_in_report = TRUE
+	report_message = "Due to the distance to our normal supply lines, cargo orders are more expensive."
+	blacklist = list(/datum/station_trait/strong_supply_lines)
+
+/datum/station_trait/distant_supply_lines/on_round_start()
+	SSeconomy.pack_price_modifier *= 1.2
+
+/datum/station_trait/late_arrivals
+	name = "Late Arrivals"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 2
+	show_in_report = TRUE
+	report_message = "Sorry for that, we didn't expect to fly into that vomiting goose while bringing you to your new station."
+	trait_to_give = STATION_TRAIT_LATE_ARRIVALS
+	blacklist = list(/datum/station_trait/random_spawns)
+
+/datum/station_trait/random_spawns
+	name = "Drive-by landing"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 2
+	show_in_report = TRUE
+	report_message = "Sorry for that, we missed your station by a few miles, so we just launched you towards your station in pods. Hope you don't mind!"
+	trait_to_give = STATION_TRAIT_RANDOM_ARRIVALS
+	blacklist = list(/datum/station_trait/late_arrivals)
+
+/datum/station_trait/blackout
+	name = "Blackout"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Station lights seem to be damaged, be safe when starting your shift today."
+
+/datum/station_trait/blackout/on_round_start()
+	. = ..()
+	for(var/a in GLOB.apcs_list)
+		var/obj/machinery/power/apc/current_apc = a
+		if(prob(30))
+			current_apc.overload_lighting()
```

Comment:
This feels like a lag trap, but it's at roundstart I guess

Reply:
we have random events that do this